doc/source/cluster/metrics.md

```diff
@@ -0,0 +1,172 @@
+(collect-metrics)=
+# Collecting metrics
+Metrics (both system and application metrics) are useful for monitoring and troubleshooting Ray applications and system. For example, you may want to access a node's metrics if it dies unexpectedly.
+
+Similar to Kubenetes, Ray records and emits time-series metrics in [Prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/). Ray does not provide a native storage solution for metrics. Users need to manage the lifecycle of the metrics by themselves. This page provides instructions on how to collect metrics from Ray clusters.
+
+
+## System and applcication metrics
+**System metrics**: Ray exports a number of system metrics. View {ref}`system metrics <system-metrics>` for more details of the emitted metrics .
+
+**Application metrics**: Application-specific metrics are useful to monitor your application states. View {ref}`adding application metrics <application-level-metrics>` for how to record your metrics.
+
+(prometheus-setup)=
+## Setting up your Prometheus server
+Ray doesn't start Prometheus servers for users. Users need to decide where to host it and configure it so that it can scrape the metrics from clusters.
+
+```{admonition} Tip
+:class: tip
+The instructions below describe how to set up Prometheus on your local machine. View [Prometheus documentation](https://prometheus.io/docs/introduction/overview/) for the best strategy to set up your Prometheus Server (e.g., whether to use a multi-tenant Prometheus instance).
+```
+
+First, [download Prometheus](https://prometheus.io/download/). Make sure to download the correct binary for your operating system. (Ex: darwin for mac osx)
+
+Then, unzip the archive into a local directory using the following command.
+
+```bash
+tar xvfz prometheus-*.tar.gz
+cd prometheus-*
+```
+Ray exports metrics only when ``ray[default]`` is installed.
```

Comment:
```suggestion
Ray exports metrics only when you install ``ray[default]``.
```

Reply:
@rkooo567 Do you know what component is needed for exporting metrics? Dashboard component? Asking this because if it is the case, other commands that install dashboard should work too.

Reply:
How about ray[air] or other installation commands that include dashboard?

Reply:
Users may use ray air or only one of the libraries. What people need to know is what we they need to do to make sure metrics can be exported. Only listing ray[default] is not enough.

```
Ray exports metrics when you use `ray[default]`, `ray[air]`, or {ref}`other installation commands <installation>`that install the Dashboard component.
```
I want something like this but I'm not sure if it's accurate or not.